packages/vite/src/index.ts

```diff
@@ -65,26 +65,24 @@ export default function kumaUI(): Plugin {
       const cssRelativePath = path
         .relative(process.cwd(), cssFilename)
         .replace(/\\/g, path.posix.sep);
-      const cssId = `/${cssRelativePath}`;
       // const css = sheet.getCSS();
       const css =
         ((result.metadata as unknown as { css: string }).css as string) || "";
-      cssLookup[cssFilename] = css;
```

Comment:
I have removed the code as I deemed it unnecessary, but please let me know if you have any concerns.